using-mobile-extensions/mobile-core/identity/identity-api-reference.md

```diff
@@ -22,64 +25,73 @@ public static void syncIdentifier(final String identifierType,
+
+o  *authenticationState* indicates the authentication state of the user, which contains one of the VisitorID.AuthenticationState values `VisitorID.AuthenticationState.AUTHENTICATED`, `VisitorID.AuthenticationState.LOGGED_OUT`, `VisitorID.AuthenticationState.UNKNOWN`
+
 #### **Example**
 
 ```java
-Identity.syncIdentifier("idType", "idValue", VisitorID.AuthenticationState.AUTHENTICATED);
+Identity.syncIdentifier("idType", 
+                        "idValue", 
+                        VisitorID.AuthenticationState.AUTHENTICATED);
 ```
 
-### syncIdentifiers
+### syncIdentifiers<a id="syncIdentifiers-java"></a>
 
-{% hint style="info" %}
-The `identifiers` map contains IDs with the Identifier type as the key, and the string identifier as the value. If the identifier type and identifier do not have valid string values \(non null or empty\), the identifier is ignored by the Identity extension.
-{% endhint %}
+The function of the `syncIdentifiers`  API is same as the `syncIdentifier` API, which passes a list of identifiers each containing an identifier type as the key and an identifier as the value. In each of the identifiers pair, both identifier type and identifier should be non empty and non null values, otherwise they will be ignore. 
 
 #### **Syntax**
 
 ```java
 public static void syncIdentifiers(final Map<String, String> identifiers,
-                                       final VisitorID.AuthenticationState authenticationState)
+                                   final VisitorID.AuthenticationState authState)
 ```
 
-#### **Example**
+o  *identifiers* map contains IDs with the Identifier type as the key, and the string identifier as the value. If the identifier type and identifier do not have valid string values \(non null or empty\), that identifier is ignored by the Identity extension.
+
+o  *authState* indicates the authentication state for the user, which contains one of the VisitorID.AuthenticationState values `VisitorID.AuthenticationState.AUTHENTICATED`, `VisitorID.AuthenticationState.LOGGED_OUT`, `VisitorID.AuthenticationState.UNKNOWN`
+
+#### Example
 
 ```java
 Map<String, String> identifiers = new HashMap<String, String>();
-identifiers.put("idType", "idValue");
+identifiers.put("idType1", "idValue1");
+identifiers.put("idType2", "idValue2");
+identifiers.put("idType3", "idValue3");
 Identity.syncIdentifier(identifiers, VisitorID.AuthenticationState.AUTHENTICATED);
 ```
 
 ### syncIdentifiers \(overloaded\)
 
-**Tip**: All given customer IDs are given the default authentication state of `UNKNOWN`.
+This  API  `syncIdentifiers()` is an overloaded version that does not include the parameter for the Authentication State, and assumes a default value of `VisitorID.AuthenticationState.UNKNOWN`
```

Comment:
- Please reword the beginning of the sentence to read "This API is an..."
- Please replace "that" with ", which"
- Please add a period at the end of the sentence.